cmake/checks/check_01_cxx_features.cmake

```diff
@@ -463,3 +468,64 @@ CHECK_CXX_SOURCE_COMPILES(
   "
   DEAL_II_HAVE_ISFINITE)
 
+
+#
+# Check that we can use feenableexcept through the C++11 header file cfenv:
```

Comment:
This isn't really the point (and I'm ok with the change) but we could also have used the fenv.h header file which predates C++11 by quite some time. What I'm checking for is really the availability of the `feenableexcept` function (i.e., a system feature), not the fact that it is supported by the compiler's C++11 support library (a compiler feature). In the long run, it doesn't matter, of course, since we will simply always assume that we have C++11.